doc/source/architecture.rst

```diff
@@ -33,8 +33,8 @@ How to set up an aggregator
 
 1. Install Libreant. Follow the instructions on :ref:`dev-installation`.
 
-2. Launch Libreant setting the ``LIBREANT_AGHERANT_DESCRIPTIONS`` environment variable.
-   Its value should be a list of URLs encoded in JSON. Each URL represents the Opensearch 
+2. Launch Libreant setting the ``AGHERANT_DESCRIPTIONS`` configuration parameters.
```

Comment:
you really mean there is no prefix anymore on environment variables? I think that this is bad.
environment variables are naturally "inherited" from processes, so prefix makes sense to avoid that setting an option for one process will also override something ele.

So if some program that we are not aware of uses an environment variables called, for example, `SYNC`, it will set such an option for libreant, too. While some conflicts are "obvious" (ie `PATH`) some other option might not be that obvious. For example we have `PORT`, which could be set for something else, too.